### PR TITLE
Flesh out process for financial aid.

### PR DESCRIPTION
**Pull request by Lukasa**
This adds some more detail on the proposed process for financial aid.

It may be worthwhile circulating this diff to the list before merging it. Up to @inglesp really.

### CODE REVIEW

File: source/financial-aid.rst [hunk 1]
```diff
@@ -68,8 +68,61 @@ It may be sensible to say that anyone who applies for financial aid, regardless
 Process
 -------
 
+Data Collection
+~~~~~~~~~~~~~~~
+
 I will be coordinating with George to formalise the data collection required for financial aid. In particular, we’ll want to set up some forms that ask the questions we believe need to be asked.
```
Comment (inglesp) [inline source/financial-aid.rst:74]: I wonder whether we can do this all through ti.to.

File: source/financial-aid.rst [hunk 2]
```diff
@@ -68,8 +68,61 @@ It may be sensible to say that anyone who applies for financial aid, regardless
...
 
+Proposed Questions
+^^^^^^^^^^^^^^^^^^
+
+The proposed questions are below, including the type of answer that can be provided, as will the purpose of the question.
+
++----------------------------+----------------+---------------------------------------------+
+| Question                   | Answer Type    | Purpose                                     |
++============================+================+=============================================+
+| Legal Name                 | Free text      | For legal reasons, this is data we need.    |
```
Comment (inglesp) [inline source/financial-aid.rst]: What are these legal reasons?

File: source/financial-aid.rst [hunk 3]
```diff
@@ -68,8 +68,61 @@ It may be sensible to say that anyone who applies for financial aid, regardless
...
+|                            |                | ticket but the delegate cannot afford one   |
+|                            |                | without aid, we may need to 'reserve' a     |
+|                            |                | ticket until the grant is processed.        |
++----------------------------+----------------+---------------------------------------------+
+| Applied to speak?          | Boolean/Number | If they have applied to speak, we'd like to |
+|                            |                | know in order to support them. We should    |
+|                            |                | aim to have a 'proposal ID' the delegate    |
+|                            |                | can provide in this field, as well.         |
++----------------------------+----------------+---------------------------------------------+
+| Planned attendance days.   | Date           | If an attendee is planning to attend only   |
```
Comment (inglesp) [inline source/financial-aid.rst]: Stray full stop!

File: source/financial-aid.rst (hunk 2 above)
Comment (Lukasa) [inline source/financial-aid.rst]: Good question. I'm being defensive here, but some possibilities include cheques etc.

Comment (inglesp) [inline source/financial-aid.rst]: If we're going to need it, we can ask for it later.  Feels potentially a little intimidating to me.

**Comment by doismellburning** [comment]
<3

File: source/financial-aid.rst (hunk 2 above)
Comment (Lukasa) [inline source/financial-aid.rst]: Ok, I'll remove. =)

**Comment by Lukasa** [comment]
Ok, feedback addressed! =)

**Comment by doismellburning** [comment]
A++ Would Merge

**Pull request merged**